tests/cpu/GpuShader_tests.cpp

```diff
@@ -1138,21 +1138,33 @@ OCIO_ADD_TEST(GpuShader, MetalSupport9)
 struct ocioOCIOMain
 {
 ocioOCIOMain(
-  int ocio_grading_rgbcurve_knotsOffsets[8]
-  , float ocio_grading_rgbcurve_knots[60]
-  , int ocio_grading_rgbcurve_coefsOffsets[8]
-  , float ocio_grading_rgbcurve_coefs[180]
+  constant int ocio_grading_rgbcurve_knotsOffsets[8]
+  , int ocio_grading_rgbcurve_knotsOffsets_count
```

Comment:
Because in current use cases, we don't fill all elements of this array with constant/uniform buffer data necessarily. Therefore, we only set it partially and set the rest to zero.